packages/app-store/office365video/lib/VideoApiAdapter.ts

```diff
@@ -52,7 +52,7 @@ const o365Auth = async (credential: CredentialPayload) => {
   if (!client_id) throw new HttpError({ statusCode: 400, message: "MS teams client_id missing." });
   if (!client_secret) throw new HttpError({ statusCode: 400, message: "MS teams client_secret missing." });
 
-  const isExpired = (expiryDate: number) => expiryDate < Math.round(+new Date() / 1000);
+  const isExpired = (expiryDate: number) => expiryDate < Math.round(+new Date());
```

Comment:
Expiry date and and `new Date()` return Unix time codes with milliseconds. We were comparing the expiry time to the current date but translated it to just seconds so `isExpired` always returned false.